core/src/main/java/org/mule/api/lifecycle/LifecycleUtils.java

```diff
@@ -15,13 +15,21 @@
 
 import org.apache.commons.collections.CollectionUtils;
 import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public abstract class LifecycleUtils
 {
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(LifecycleUtils.class);
+
+    public static void initialiseIfNeeded(Object object) throws InitialisationException
+    {
+        initialiseIfNeeded(object, null);
+    }
+
     public static void initialiseIfNeeded(Object object, MuleContext muleContext) throws InitialisationException
     {
-        if (object instanceof MuleContextAware)
+        if (muleContext != null && object instanceof MuleContextAware)
```

Comment:
because I added another signature for this method in which you don't pass a muleContext because you don't consider the object being MuleContextAware... In order to avoid code repetition,  I added that check so that the new method can delegate in this one